TriblerGUI/dialogs/feedbackdialog.py

```diff
@@ -58,13 +61,20 @@ def add_item_to_info_widget(key, value):
         for key in os.environ.keys():
             add_item_to_info_widget('os.environ', '%s: %s' % (key, os.environ[key]))
 
+        # Add recent requests to feedback dialog
         request_ind = 1
         for endpoint, method, data, timestamp, status_code in sorted(tribler_performed_requests.values(),
                                                                      key=lambda x: x[3])[-30:]:
             add_item_to_info_widget('request_%d' % request_ind, '%s %s %s (time: %s, code: %s)' %
                                     (endpoint, method, data, timestamp, status_code))
             request_ind += 1
 
+        # Add recent events to feedback dialog
+        events_ind = 1
+        for event, event_time in received_events[:30][::-1]:
```

Comment:
Shouldn't this be received_events[-30:][::-1]?

Reply:
@qstokkink no, received_events are ordered by time, descending. So this line takes the 30 most recent events and reverses them 👍